wpilibj/build.gradle

```diff
@@ -8,10 +8,22 @@ sourceSets {
 dependencies {
     sharedCompile 'edu.wpi.first.wpilib.networktables.java:NetworkTables:3.0.0-SNAPSHOT:arm'
     sharedRuntime 'edu.wpi.first.wpilib.networktables.java:NetworkTables:3.0.0-SNAPSHOT:arm'
+    testCompile group: 'org.hamcrest', name: 'hamcrest-all', version: '1.3'
```

Comment:
Please follow the same conventions the rest of the gradle files follow, ie the same as the line above.